src/parse_logcfg.c

```diff
@@ -1036,47 +1036,47 @@ static config_t logcfg_configs[] = {
     {"USEPARTIALS",     CFG_INT_ONE(use_part)},
```

Comment:
Definitely improves readability in my opinion.
Practically every `CFG_INT_ONE` config is a bool.

Reply:
True, there are quite some more changes to do. Will do it in next days.